docs/source/app_developers_guide/go_sdk_import.rst

```diff
@@ -0,0 +1,59 @@
+********************
+Importing the Go SDK
+********************
+
+.. note::
+   The Sawtooth Go SDK assumes that you have the latest version of Go,
+   more information on installing and configuring can be found at `go
+   <https://github.com/golang/go>`_.
+
+Once you've got a working version of Sawtooth, there are a few additional
+steps you'll need to take to get started developing for Sawtooth in Go.
+
+1. Get Go SDK located at `Sawtooth Go SDK repository
+   <https://github.com/hyperledger/sawtooth-sdk-go>`_.
+
+.. code-block:: console
+    :caption: Run following command for downloading Go SDK
+
+    $ go get github.com/hyperledger/sawtooth-sdk-go
+
+2. Import the SDK into your Go files. You need to specify packages from SDK
+   which are needed.
+
+.. code-block:: go
+    :caption: Sample header for a Go file
+
+    import (
+        // to use signing package from SDK
+        "github.com/hyperledger/sawtooth-sdk-go/signing"
+    )
+
+    // --snip--
+
+3. Generate protobuf files for use, go to location
+   ``$GOPATH/src/github.com/hyperledger/sawtooth-sdk-go`` and run ``go generate``
```

Comment:
Also, is `$GOPATH` automatically set for all users?  Or does it need to be explained here?